### PR TITLE
Added maintenance updates up to 15.2.10 (#911)

### LINKED ISSUE #911: SES7: add 15.2.6, .7, .8, .9, and .10 notable changes to Appendix A
**Link To Section**

https://documentation.suse.com/en-us/ses/7/single-html/ses-deployment/#id-1.3.7

**Summary**

I noticed a bunch of upstream Octopus point releases are missing from Appendix A. Time to update it?

**Notes**

15.2.6 https://ceph.io/community/v15-2-6-octopus-released/
15.2.7 https://ceph.io/community/v15-2-7-octopus-released/
15.2.8 https://ceph.io/community/v15-2-8-octopus-released/
15.2.9 https://ceph.io/community/v15-2-9-octopus-released/
15.2.10 https://ceph.io/community/v15-2-10-octopus-released/

### PR DESCRIPTION
**Pull request by tbazant**
closes #911

### CODE REVIEW

File: xml/ceph_maintenance_updates.xml [hunk 1]
```diff
@@ -25,6 +25,123 @@
...
   product.
  </para>
+ <bridgehead renderas="sect1">&cephname; 15.2.10 Point Release</bridgehead>
+ <para>
+  This backport release includes the following fixes:
+ </para>
+ <itemizedlist>
+  <listitem>
+   <para>
+    The containers include an updated tcmalloc that avoids crashes seen on 15.2.9.
```
Comment (asettle) [inline xml/ceph_maintenance_updates.xml:35]:
```suggestion
    The containers include an updated <literal>tcmalloc</literal> that avoids crashes seen on 15.2.9.

File: xml/ceph_maintenance_updates.xml [hunk 2]
```diff
@@ -25,6 +25,123 @@
...
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    RADOS: &bluestore; handling of huge (>4GB) writes from RocksDB to BlueFS
```
Comment (asettle) [inline xml/ceph_maintenance_updates.xml:40]: Do we need to use the `&lt;` here instead?

**Review by asettle — APPROVED** [review]

File: xml/ceph_maintenance_updates.xml [hunk 3]
```diff
@@ -25,6 +25,123 @@
...
+ </para>
+ <itemizedlist>
+  <listitem>
+   <para>
+    The containers include an updated <literal>tcmalloc</literal> that avoids crashes seen on 15.2.9.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    RADOS: &bluestore; handling of huge (>4GB) writes from RocksDB to BlueFS
```
Comment (tbazant) [inline xml/ceph_maintenance_updates.xml:40]: good point (although this validates, it's unclean)
```suggestion
    RADOS: &bluestore; handling of huge (&gt;4GB) writes from RocksDB to BlueFS
```

**Comment by smithfarm** [comment]
Would it make sense to give @callithea the ability to review/approve PRs in this repo?

**Review by callithea — APPROVED** [review]

**Pull request merged**